core/example/longrange/ewald.cpp

```diff
@@ -103,210 +76,428 @@ double TEwald::compute( ParticleList &particles, double lx, double ly,
     auto r = Cabana::slice<Position>( particles );
     auto q = Cabana::slice<Charge>( particles );
     auto p = Cabana::slice<Potential>( particles );
+    auto f = Cabana::slice<Force>( particles );
+    auto v = Cabana::slice<Velocity>( particles );
+    auto i = Cabana::slice<Index>( particles );
 
     int n_max = particles.size();
 
-    auto init_p = KOKKOS_LAMBDA( const int idx ) { p( idx ) = 0.0; };
+    auto init_p = KOKKOS_LAMBDA( const int idx )
```

Comment:
If we init more than p, maybe we should call it something else?

Reply:
Renamed to init_parameters (unless you have a better name?)